groupcache_test.go

```diff
@@ -217,6 +259,12 @@ func countFills(f func()) int64 {
 	return cacheFills.Get() - fills0
 }
 
+func countMeta(f func()) int64 {
```

Comment:
unused - is this useful to add somewhere? Seems like we updated the existing interfaces with Container but aren't explicitly testing it anywhere yet?

Reply:
I did add it, don't remember why I removed it. I just added it back

Reply:
Lmk if more testing is needed.